projects/go-tester/src/app/app.component.html

```diff
@@ -1,110 +1,13 @@
-              <go-loader #loader loaderSize="small"></go-loader>
-              <go-loader></go-loader>
-              <go-loader loaderSize="large"></go-loader>
-            </div>
-          </div>
-        </div>
-        
-        <div class="go-column go-column--100">
-          <h4>Buttons</h4>
-          <go-button buttonIcon="home">
-            Hey
-          </go-button>
-          <go-button buttonIcon="add_shopping_cart" buttonVariant="positive" (handleClick)="clickHey()" [useLoader]="true" #heyButton>
-            Hey
-          </go-button>
-          <go-button buttonIcon="fingerprint" buttonVariant="negative">
-            Hey
-          </go-button>
-          <go-button buttonIcon="fingerprint" buttonVariant="negative" buttonDisabled="true">
-            Nope
-          </go-button>
-          <go-button buttonIcon="plus_one" buttonVariant="neutral">
-            Hey
-          </go-button>
-          <go-button [buttonIcon]="'home'" buttonVariant="positive" [useDarkTheme]="true">
-            Dark
-          </go-button>
-        </div>
-        <div class="go-column go-column--100">
-          <h4>Icon Button</h4>
-          <go-icon-button buttonIcon="home" (handleClick)="openThing()"></go-icon-button>
-        </div>
-
-        <div class="go-column go-column--100">
-          <h4>Off Canvas</h4>
-          <go-button buttonIcon="home" (handleClick)="openThing()">
-            Click me to open
-          </go-button>
-        </div>
-
-        <div class="go-column go-column--100">
-          <h4>Toasts</h4>
-          <go-toast type="neutral"
-                    header="Announcement!"
-                    [dismissable]="true">
-          </go-toast>
-        </div>
-
-        <div class="go-column go-column--100">
-          <h4>Table</h4>
-          <go-table [loadingData]="tableLoading"
-                    [tableConfig]="tableConfig"
-                    (tableChange)="handleTableChange($event)"
-                    tableTitle="People Data"
-                    *ngIf="tableConfig">
-            <go-table-column field="id" title="ID"></go-table-column>
-            <go-table-column field="name.first" title="First Name"></go-table-column>
-            <go-table-column field="name.last" title="Last Name"></go-table-column>
-            <go-table-column field="email" title="Email"></go-table-column>
-            <go-table-column field="gender" title="Gender"></go-table-column>
-            <go-table-column field="ip_address" title="IP Address"></go-table-column>
-            <go-table-column>
-              <ng-template #goTableCell let-item>
-                <go-icon icon="delete"></go-icon>
-              </ng-template>
-            </go-table-column>
-          </go-table>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-<go-toaster></go-toaster>
-<go-off-canvas></go-off-canvas>
+<go-layout>
+  <go-header go-layout-header [logo]="logo">
+    <go-search go-header-middle>
```

Comment:
Are we going to run into problems here if a user doesn't have permission to view the search bar in the middle? Essentially if you put an ng-if on this go-header-middle will the header still look correct?

Reply:
@StevenUlmer is that a thing? There are users that can't see the search bar? That being said, if the `go-layout` is rendered without a `go-header-middle` the header will just render with a blank spot in the middle like so:
![Screen Shot 2019-07-24 at 8 57 00 AM](https://user-images.githubusercontent.com/11966331/61795293-0e4a8880-adf1-11e9-8354-d10125bbf446.png)

Reply:
👍 ya there's actually a large part of our users who can't see the search as they don't have permission to do anything with anyone besides themselves. This looks great tho without it 🎉